src/main/java/org/gridsuite/network/map/dto/mapper/GeneratorInfosMapper.java

```diff
@@ -129,22 +129,22 @@ private static GeneratorFormInfos toFormInfos(Identifiable<?> identifiable) {
                 .generatorStartup(toGeneratorStartup(generator))
                 .coordinatedReactiveControl(toCoordinatedReactiveControl(generator));
 
-        Terminal regulatingTerminal = generator.getRegulatingTerminal();
+        Terminal regulatingTerminalForm = generator.getRegulatingTerminal();
```

Comment:
Why this renaming ? Why does `Form` means here ?

Reply:
I renamed it to avoid duplicated code because I had 30% of duplication for this part of code